<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>P2PPage</class>
 <widget class="QWidget" name="P2PPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>390</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>390</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>48</y>
     <width>661</width>
     <height>327</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="verticalSpacing">
     <number>3</number>
    </property>
    <item row="4" column="0">
     <widget class="QLabel" name="label_period_2">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;5. Loan period&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" alignment="Qt::AlignTop">
     <widget class="QLabel" name="label_msg_2">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;6. Message to lender&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_amount_2">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;2. Amount to borrow&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="bSubmit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Find lenders</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="le_address">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="cb_premium">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>1%</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3%</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5%</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7%</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10%</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>25%</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>50%</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>100%</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_premium_2">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;3. Select premium&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_address_2">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;1. Your UNIQ address for loan&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_expiry_2">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;4. Loan request expiry&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="le_amount">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="cb_period">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>1 week</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2 weeks</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1 month</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3 months</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6 months</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1 year</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="cb_expiry">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>1 day</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1 week</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1 month</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QTextEdit" name="te_message">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>60</height>
       </size>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;(optional)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_pwd">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>220</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;7. Wallet passphrase&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="le_pwd">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>30</height>
       </size>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_sub">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>220</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;8. Submit&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>491</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Complete the following form to submit loan request to compatible lenders:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
